web-api/src/app.ts

```diff
@@ -800,6 +801,10 @@ app.get(
     '/judge-activity-report/closed-cases',
     lambdaWrapper(getCasesClosedByJudgeLambda),
   );
+  app.post(
+    '/async/reports/csv/custom-case-report',
+    lambdaWrapper(createCsvCustomCaseReportFileLambda, { isAsync: true }),
+  );
```

Comment:
somewhat nitpicky, but could this be rearranged please?

`/async/reports/custom-case-report/csv`

I'd also consider adding format=csv into a request variable

Reply:
I also think we should probably add the keyword "export" in the path .
Maybe 
`/async/export/reports/custom-case-report/csv`

Using a request variable can probably be done when we start supporting multiple formats, not now.